src/resources/breaking-changes.md

```diff
@@ -0,0 +1,821 @@
+---
+title: Breaking changes and deprecations
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists all language and library breaking changes and deprecations in Dart,
+organized by release and area, to help Dart users understand and manage their
+impact. Complete release notes are available in the [Dart SDK changelog][changelog].
+The [breaking change policy][] document describes the policy and process
+around breaking changes and deprecations in Dart. 
+
+**This page includes two types of breaking changes, as well as deprecations**:
+* **Unversioned**: The compiler does not maintain backward compatibility, and
+  code may break as soon as you [upgrade your sdk version][sdk] if it relies on
+  the previous behavior.
+  
+  *These are the majority of changes and are not specially marked in this list.*
+* **Language versioned**: The compiler maintains backward compatibility for
+  existing code, and the behavior change only takes effect (potentially breaking
+  code that relies on the previous behavior) when you upgrade the
+  [language version] of your code.
+  
+  *These are marked "**Language versioned**"*.
+* **Deprecations**: The compiler maintains compatibility for deprecated code,
+  with a warning. Deprecations are then completely removed in the release
+  following their deprecation, breaking any code that relies on the previous behavior.
+  
+  *These are marked "**Deprecated**" and "**Removed**", respectively*.
+
+If you have questions or concerns about any of these breaking changes, please 
+comment on the breaking change issue linked from the relevant entry.
+To be notified about future breaking changes, join the [Dart announce][] group.
+
+[breaking change policy]: https://github.com/dart-lang/sdk/blob/main/docs/process/breaking-changes.md
+[changelog]: https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md
+[sdk]: /get-dart
+[language version]: /guides/language/evolution#language-versioning
+[Dart announce]: https://groups.google.com/a/dartlang.org/g/announce
+
+## Not yet released to stable
```

Comment:
Just a note to update this for the 3.1 release. Note that the changelog may have updated since you wrote this as well.

Reply:
About 3.1: Do you think [the removal of `ObjectLiteral`](https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#dartjs_interop-1) is a breaking change or should be marked "Removed" under the deprecation format I just added to this PR?

Reply:
Good question. That's tough since `dart:js_interop` is WIP and comes with a warning that it may change, so its not a traditional breaking change since that status allows for it.

I guess it's fine to just call it a breaking change even though any users should be prepared for the break.